src/panels/lovelace/special-rows/hui-button-row.ts

```diff
@@ -21,7 +21,7 @@ import { handleAction } from "../common/handle-action";
 @customElement("hui-button-row")
 export class HuiButtonRow extends LitElement implements LovelaceRow {
   public hass?: HomeAssistant;
-  @property() private _config?: ButtonRowConfig;
+  @property() protected _config?: ButtonRowConfig;
```

Comment:
I dont think you need to do this. Should be able just call `super.setConfig(config)`

Reply:
good call